src/scripts.js

```diff
@@ -5,8 +5,13 @@ import './images/turing-logo.png';
 import Recipe from './classes/Recipe';
 import RecipeRepository from './classes/RecipeRepository';
 import recipeData from './data/recipes';
+import User from './classes/User';
+import usersData from './data/users';
 // ------------------- GLOBAL VARIABLES
 const recipeRepo = new RecipeRepository(recipeData);
```

Comment:
I think we need to change this to the new wya to state the recipe repo - `const recipeRepo = RecipeRepository.fromRecipeData(recipeData)`